luigi/contrib/hdfs/target.py

```diff
@@ -214,7 +214,7 @@ def __init__(self, path, format=None, client=None, flag='_SUCCESS'):
         if path[-1] != "/":
             raise ValueError("HdfsFlagTarget requires the path to be to a "
                              "directory.  It must end with a slash ( / ).")
-        super(HdfsFlagTarget, self).__init__(path, format, client)
+        super(HdfsFlagTarget, self).__init__(path, format, is_tmp=False, fs=client)
```

Comment:
There is not need to set `is_tmp=False` which is the default value in `HdfsTarget`.

Reply:
I thought that for sake of human readability I better should name-mention parameters that seem being out of context / have different naming but if it doesn't fit, I'll change that

Reply:
I meant there is no need to pass in is_tmp, fs=client is sufficient.

Reply:
Agreed, fixed